lstchain/io/event_selection.py

```diff
@@ -43,10 +46,19 @@ class EventSelector(Component):
 
     def filter_cut(self, events):
         """
-        Apply the event filters
+        Apply the standard event filters.
         """
         return filter_events(events, self.filters, self.finite_params)
 
+    def same_disp_sign(self, events):
+        """
+        Apply a mask to have events with the reconstructed disp sign,
+        be the same as the true, to avoid wrong head-tail assignment
+        in the reconstructed image of the MC data.
```

Comment:
docstring of input/output would also be useful

Reply:
Will the change in the docstring be sufficient?